plugin/parser/differ/mysql/differ.go

```diff
@@ -105,5 +106,35 @@ func buildTableMap(nodes []ast.StmtNode) map[string]*ast.CreateTableStmt {
 
 // isTwoColumnsSame returns true if the two columns are the same.
 func isTwoColumnsSame(old, new *ast.ColumnDef) bool {
-	return old.Tp.String() == new.Tp.String()
+	if old.Tp.String() != new.Tp.String() {
+		return false
+	}
+	oldNormalizeOptions := normalizeOptions(old.Options)
+	newNormalizeOptions := normalizeOptions(new.Options)
+	if len(oldNormalizeOptions) != len(newNormalizeOptions) {
+		return false
+	}
+	for idx, oldOption := range oldNormalizeOptions {
+		newOption := newNormalizeOptions[idx]
+		if oldOption.Tp != newOption.Tp {
```

Comment:
For some column options, only comparing the Type is not enough.
Please compare the whole option or add a TODO comment now.

Reply:
Right. I will add the `switch-case` struct to handle that. But it's not need for this PR which only handle the NULL/NOTNULL option.

Reply:
So, add the TODO comment at least.

Reply:
Done.